.github/workflows/auto-release.yml

```
@@ -0,0 +1,48 @@
+name: Automated Releases
+on: workflow_dispatch
+
+# This Workflow is triggered through the GitHub API:
+#   curl -X Post \
+#        -H "Authorization: Bearer <token>" \
+#        -d '{"ref":"master"}' \
+#        https://api.github.com/repos/simple-icons/simple-icons-pdf/actions/workflows/auto-release.yml/dispatches
+# Replacing <token> by a personal access token with scope `public_repo`
+
+jobs:
+  auto-release:
+    name: Automated release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # Ensure we are checked out on the master branch
+          ref: master
+          # Ensure custom credentials are used when pushing
+          persist-credentials: false
+      - name: Update simple-icons
+        id: update
+        run: |
+          newVersion="$(node scripts/bump-simpleicons.js)"
+          echo ::set-output name=NEW_VERSION::$newVersion
+      - name: Regenerate icons
+        run: |
+          # Delete Existing PDFs
+          npm run clean
+          # Generate New PDFs
+          npm run build
+      - name: Commit updates
+        run: |
+          # Set up git credential
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git config --global user.name "github-actions[bot]"
+          # Create a commit for the simple-icons dependency update
+          git add .
+          git commit -m "Update simple-icons to ${{ steps.update.outputs.NEW_VERSION }}"
+      - name: Push updates
+        run: |
+          # Set up remote using a Personal Access Token
+          git remote remove origin
+          git remote add origin https://${{secrets.RELEASE_TOKEN}}@github.com/simple-icons/simple-icons-pdf.git
+          # Push updates to origin
```

I would prefer this with the extra enter:

```suggestion
          git remote add origin https://${{secrets.RELEASE_TOKEN}}@github.com/simple-icons/simple-icons-pdf.git

          # Push updates to origin
```

Alternatively, we could split this up into two steps, "Set remote" and "Push updates" but then we _should_ update it in every repository.

Either solution works for me.